add guide for basic rbac (#557)
clerk/clerk-docs: docs/backend-requests/making/custom-session-token.mdx

```
</CodeBlockTabs>

## Add global TypeScript type for additional session claims

> For projects using TypeScript, a global type for the additional session claims will avoid type errors in your code and provide auto-completion in your editior.

Reviewer: TypeScript is mentioned in the Header
```suggestion
A global type for additional session claims defined in a declaration file avoids type errors and provides auto-completion.
```
